docs-v2/content/en/docs/pipeline-stages/deployers/cloudrun.md

```diff
@@ -6,7 +6,7 @@ featureId: deploy.cloudrun
 ---
 
 {{< alert title="Note" >}}
-This feature is currently experimental and subject to change. Not all Skaffold features e.g. log tailing, debugging are supported.
+This feature is currently experimental and subject to change. Not all Skaffold features e.g. debugging are supported.
```

Comment:
nit: Can you change this from:

Not all Skaffold features e.g. debugging are supported.
TO

Not all Skaffold features are supported, for example `debug` is currently not support in Cloud Run (but is on our Roadmap)

Reply:
Done